crowbar_framework/app/models/provisioner_service.rb

```diff
@@ -179,76 +179,85 @@ def transition(inst, name, state)
   end
 
   def load_repository_bag
-    bag = Chef::DataBag.load("repositories") rescue nil
+    bag = Chef::DataBag.load("crowbar/repositories") rescue nil
     if bag.nil?
-      create_repository_bag
+      @logger.debug("Creating repositories data bag")
+      bag = Chef::DataBagItem.new
+      bag.data_bag "crowbar"
+      bag["id"] = "repositories"
+      bag.save
     end
     bag
   end
 
   def synchronize_repositories(platforms)
-    # create DataBag if it doesn't exist yet
-    load_repository_bag
+    bag = load_repository_bag
+
     platforms.each do |platform, repos|
-      repos.each do |reponame, active|
-        repo_object = Crowbar::Repository.where(platform: platform, repo: reponame).first
+      repos.each do |repo, active|
+        repo_object = Crowbar::Repository.where(platform: platform, repo: repo).first
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `repo_object`. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)